rust/index/src/fulltext/types.rs

```diff
@@ -56,7 +56,7 @@ pub struct FullTextIndexWriter<'me> {
     full_text_index_reader: Option<FullTextIndexReader<'me>>,
     posting_lists_blockfile_writer: BlockfileWriter,
     frequencies_blockfile_writer: BlockfileWriter,
-    tokenizer: Arc<Mutex<Box<dyn ChromaTokenizer>>>,
+    tokenizer: Arc<Box<dyn ChromaTokenizer>>,
```

Comment:
can this just be `Arc < dyn ... >`

Reply:
that would then require callers to provide `Arc<dyn ...>` to the constructor, which seems a little leaky
but not a strong opinion, fine either way

Reply:
no strong opinion